bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -295,20 +305,6 @@ case object SemesterConverter2014BTo2015A extends SemesterConverter {
   }
   override val transformers = List(gsSubmission)
 }
-/**
- * This converter only changes the schema version but retains the semester
- */
-case object SchemaVersionConverter extends SemesterConverter {
```

Comment:
We shouldn't delete this. even if it is unused, it is useful when we have bugfix releases inside the same semester